nuxeo/values.yaml

```diff
@@ -61,6 +61,7 @@ nuxeo:
   ##packages: nuxeo-web-ui
   custom_params: |-
     # add any other params that go into nuxeo.conf
+    elasticsearch.indexNumberOfReplicas=0
```

Comment:
Not sure we can declare it here, this value will be seen as the default and won't be appended if someone defines `custom_param`, for instance on nuxeo-preview:
```bash
$ kubectl -n nuxeo-preview-master get cm nuxeo-preview-conf -oyaml
apiVersion: v1
data:
  nuxeo.conf: |-
    # Additional nuxeo.conf parameters
    nuxeo.stream.work.log.codec=avro
    nuxeo.stream.audit.log.codec=avro
    nuxeo.stream.pubsub.log.codec=avro
    nuxeo.templates=default,mongodb
    nuxeo.mongodb.server=mongodb://nuxeo-preview-master-mongodb:27017
    nuxeo.mongodb.dbname=nuxeo
    elasticsearch.client=RestClient
    elasticsearch.clusterName=elasticsearch
    elasticsearch.addressList=http://nuxeo-preview-master-elasticsearch-client:9200
    elasticsearch.indexName=nuxeo
    elasticsearch.restClient.socketTimeoutMs=300000
    elasticsearch.restClient.connectionTimeoutMs=300000
kind: ConfigMap
metadata:
  annotations:
    jenkins.io/chart: preview
    kubectl.kubernetes.io/last-applied-configuration: |
      {"apiVersion":"v1","data":{"nuxeo.conf":"# Additional nuxeo.conf parameters\nnuxeo.stream.work.log.codec=avro\nnuxeo.stream.audit.log.codec=avro\nnuxeo.stream.pubsub.log.codec=avro\nnuxeo.templates=default,mongodb\nnuxeo.mongodb.server=mongodb://nuxeo-preview-master-mongodb:27017\nnuxeo.mongodb.dbname=nuxeo\nelasticsearch.client=RestClient\nelasticsearch.clusterName=elasticsearch\nelasticsearch.addressList=http://nuxeo-preview-master-elasticsearch-client:9200\nelasticsearch.indexName=nuxeo\nelasticsearch.restClient.socketTimeoutMs=300000\nelasticsearch.restClient.connectionTimeoutMs=300000"},"kind":"ConfigMap","metadata":{"annotations":{"jenkins.io/chart":"preview"},"labels":{"app":"nuxeo-preview","chart":"nuxeo-1.0.6","heritage":"Tiller","jenkins.io/chart-release":"nuxeo-preview-master","jenkins.io/namespace":"nuxeo-preview-master","jenkins.io/version":"0.1.0-SNAPSHOT","release":"nuxeo-preview-master","role":"config"},"name":"nuxeo-preview-conf","namespace":"nuxeo-preview-master"}}
  creationTimestamp: "2020-04-08T15:35:21Z"
  labels:
    app: nuxeo-preview
    chart: nuxeo-1.0.6
    heritage: Tiller
    jenkins.io/chart-release: nuxeo-preview-master
    jenkins.io/namespace: nuxeo-preview-master
    jenkins.io/version: 0.1.0-SNAPSHOT
    release: nuxeo-preview-master
    role: config
  name: nuxeo-preview-conf
  namespace: nuxeo-preview-master
  resourceVersion: "177278892"
  selfLink: /api/v1/namespaces/nuxeo-preview-master/configmaps/nuxeo-preview-conf
  uid: 8ebbc4f5-79ae-11ea-85f8-42010a8e0158
```
I think this is due to `custom_params` and its String type, not sure it happens on arrays for instance.

Reply:
now goes in the configMap and defined as its own value as well as     ```
elasticsearch.restClient.socketTimeoutMs=300000
elasticsearch.restClient.connectionTimeoutMs=300000```